data/channels.csv

```diff
@@ -1502,6 +1502,7 @@ AsesinatosdeMidsomerSpain.es,Asesinatos de Midsomer Spain,,,,ES,,,c/ES,spa,serie
 AsfiyahiTV.sn,Asfiyahi TV,,,,SN,,,c/SN,fra,,FALSE,,,,https://www.asfiyahi.org/,https://i.imgur.com/tDuvfsh.png
 Asgabat.tm,Asgabat,,,,TM,,,c/TM,tuk,,FALSE,,,,http://turkmentv.gov.tm/,https://turkmentv.gov.tm/assets/img/live-imgs/ashgabat.png
 AshamTV.et,Asham TV,,,,ET,,,c/ET,amh,,FALSE,,,,http://www.ashamtv.com/,https://i.imgur.com/YEasUNg.png
+AsharqDocumentary.sa,Asharq Documentary,الشرق الوثائقية,,Saudi Research and Media Group,SA,,Riyadh,c/SA,ara,documentary,FALSE,2023-08-31,,,https://now.asharq.com/%D9%85%D8%A8%D8%A7%D8%B4%D8%B1-%D8%A7%D9%84%D8%B4%D8%B1%D9%82-%D8%A7%D9%84%D9%88%D8%AB%D8%A7%D8%A6%D9%82%D9%8A%D8%A9,https://i.imgur.com/Xrvjb2h.png

```

Comment:
Please update your request:

- the size of the logo should not exceed 512x512px

https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels

Reply:
Sorry! fixed!